### DISCUSSION TITLE
error concerned with pcigale genconf

### DISCUSSION THREAD

**Discussion by gutudgithub** [mboquien/cigale#47] [General]
I try to run pcigale.init in order to estimate stellar mass and, luminosity and star formation rate; however I face some problems concerned with that. According to the Cigale manual; After setting the initial configuration file, run the following in a terminal
                 $ pcigale genconf                                                                                                                  
   I have got the following error: "AttributeError: module 'pcigale.analysis_modules' has no attribute 'Module' ".

**Comment by mboquien** [comment]
Could you attach your `pcigale.ini` and `pcigale.ini.spec` files?
